### ISSUE TITLE
Simple build scripts

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#2635]
Should we add simple build scripts to build teeworlds, like the ones in the bam repository?

**Comment by Learath2** [comment]
It'd be far more trivial then bam's one as both with cmake and bam we just need like 2-3 commands max.

I think this would be a good idea only if we find other conveniences this script could provide. Like packaging or some output that makes development easier.

**Comment by oy** [comment]
On windows people don't need to start the `Native tool command prompt` for bam. Just click it and it builds teeworlds.

**Comment by Dune-jr** [comment]
Yeah it's not easy on Windows. Can we really detect where the `Native tool command prompt` is though? And if we can, why not put it in the bam script?

**Comment by Learath2** [comment]
There used to be ways to track down vcvarsall.bat haven't used windows in a
while so idk how it's done nowadays

**Comment by ChillerDragon** [comment]
I usually use right click -> open in visual studio on windows and there is also no need for a command prompt it just builds. In general I am a big fan of easy to use scripts but they wont help if they are not found. The root directory is full of things like license, bam, cmakelists and storage.cfg im not sure if yet another bat and shell file will make things easier.

**Comment by oy** [comment]
@Dune-jr how it's done in [bam](https://github.com/teeworlds/bam/commit/e013f96b2fdb823c365a175aa83572c5bb9f1132)
Could probably be done in the bam script that builds tw too.

**Comment by Dune-jr** [comment]
> @Dune-jr how it's done in [bam](https://github.com/teeworlds/bam/commit/e013f96b2fdb823c365a175aa83572c5bb9f1132)
> Could probably be done in the bam script that builds tw too.

Yeah, and then I don't think we would need another script, `bam` is simple enough
